core/pom.xml

```diff
@@ -224,6 +224,10 @@
       <groupId>org.scala-lang</groupId>
       <artifactId>scala-library</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-reflect</artifactId>
```

Comment:
Why is this one needed?

Reply:
This is required because, jackson-scala-module underwent this change: 

https://github.com/FasterXML/jackson-module-scala/commit/d6071ac4597e582b1a59b5a492ac38009e17eeb6#diff-fdc3abdfd754eeb24090dbd90aeec2ce

And Spark-core uses scala-reflect library at one place.

Reply:
`org.apache.spark.deploy.security.HBaseDelegationTokenProvider`